tests/benchdnn/inputs/graph/complex_fusion/generated/sdpa-dqk-scl-msk-dqv.json

```diff
@@ -0,0 +1 @@
ided", "property_type": "variable"}]}, {"id": 300, "name": "scale_s", "kind": "Divide", "attrs": {"auto_broadcast": {"type": "string", "value": "numpy"}}, "inputs": [{"id": 15, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}, {"id": 4, "dtype": "f16", "shape": [1], "stride": [1], "layout_type": "strided", "property_type": "undef"}], "outputs": [{"id": 16, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}]}, {"id": 400, "name": "mask_add", "kind": "Add", "attrs": {"auto_broadcast": {"type": "string", "value": "numpy"}}, "inputs": [{"id": 16, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}, {"id": 5, "dtype": "f16", "shape": [1, 1, 384, 384], "stride": [147456, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}], "outputs": [{"id": 18, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}]}, {"id": 500, "name": "softmax", "kind": "SoftMax", "attrs": {"axis": {"type": "s64", "value": 3}}, "inputs": [{"id": 18, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}], "outputs": [{"id": 27, "dtype": "f16", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}]}, {"id": 600, "name": "dequantize_v", "kind": "DynamicDequantize", "attrs": {"qtype": {"type": "string", "value": "per_group"}, "group_shape": {"type": "s64[]", "value": [1, 1, 1, 64]}, "axis": {"type": "s64", "value": 3}}, "inputs": [{"id": 6, "dtype": "s8", "shape": [1, 16, 384, 64], "stride": [393216, 24576, 64, 1], "layout_type": "strided", "property_type": "variable"}, {"id": 7, "dtype": "f16", "shape": [1, 16, 384, 1], "stride": [6144, 384, 1, 1], "layout_type": "strided", "property_type": "undef"}, {"id": 8, "dtype": "s8", "shape": [1, 16, 384, 1], "stride": [6144, 384, 1, 1], "layout_type": "strided", "property_type": "undef"}], "outputs": [{"id": 45, "dtype": "f16", "shape": [1, 16, 384, 64], "stride": [393216, 24576, 64, 1], "layout_type": "strided", "property_type": "variable"}]}, {"id": 700, "name": "matmul_v", "kind": "MatMul", "attrs": {"transpose_a": {"type": "bool", "value": 0}, "transpose_b": {"type": "bool", "value": 0}}, "inputs": [{"id": 27, "dtype": "f16", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}, {"id": 45, "dtype": "f16", "shape": [1, 16, 384, 64], "stride": [393216, 24576, 64, 1], "layout_type": "strided", "property_type": "variable"}], "outputs": [{"id": 50, "dtype": "f16", "shape": [1, 16, 384, 64], "stride": [393216, 24576, 64, 1], "layout_type": "strided", "property_type": "variable"}]}]}
```

Comment:
Is it similar to [sdpa-compressed-kv-int4-gs32.json](https://github.com/uxlfoundation/oneDNN/blob/main/tests/benchdnn/inputs/graph/complex_fusion/mha/sdpa-compressed-kv-int4-gs32.json)?